frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/guest-agent-info/guest-agent-info-wrapper.ts

```diff
@@ -30,6 +30,8 @@ export class GuestAgentInfoWrapper extends Wrapper<
 
```

Comment:
the constructor of this wrapper doesn't work properly. What happens when you pass copy=true? It is hard to track the source of truth + you have to synchronize changes. 

I am not against wrapper composition if you think it makes your life easier. But for example could we rather reference `this.data`  and create new instance of GuestAgentOSInfoWrapper in getOSInfo? etc.

Reply:
> the constructor of this wrapper doesn't work properly.

works for me, am I missing something ?

Reply:
right now it does because of the limited usage. But the implementation should be correct for all the functionality it offers.

Reply:
I like current pattern better, I think it's cleaner and correct.
but I don't mind the other pattern too:
`getFSInfo = (): GuestAgentFSInfoWrapper => new GuestAgentOSInfoWrapper(this.data?.os);`

Reply:
It keeps the source of truth in two places. And will cause bugs in the future when you want to modify it somehow. 

For example, `constructor(copy=true), asResource, mergeWith and omitEmpty` depend on this one source of truth. The changes will not propagate correctly.

Reply:
Done! Thanks for pointing out the error. It didn't occur to me. =)